pyproject.toml

```diff
@@ -68,9 +68,12 @@ target-version = "py38"
 known-first-party = ["vizro"]
 
 [tool.ruff.per-file-ignores]
+# Ignore current false positives for pydantic models subclassing across files
+# See: https://github.com/astral-sh/ruff/issues/5243#issuecomment-1860776975
+"**/models/**" = ["RUF012"]
 # Tests can use magic values, assertions, and relative imports, ignore missing docstrings,
 # compare to empty string
-"**/tests/**" = ["PLR2004", "S101", "TID252", "D100", "D101", "D102", "D103", "PLC1901"]
+"**/tests/**" = ["PLR2004", "S101", "TID252", "D100", "D101", "D102", "D103", "PLC1901", "RUF012"]
```

Comment:
Be aware of these ignores and check the issue comment to understand why it's ok to ignore for now